src/quo/components/avatars/token_avatar/view.cljs

```diff
@@ -0,0 +1,31 @@
+(ns quo.components.avatars.token-avatar.view
+  (:require [quo.components.avatars.token-avatar.schema :as token-avatar.schema]
+            [quo.components.avatars.token-avatar.style :as style]
+            [react-native.core :as rn]
+            [react-native.hole-view :as hole-view]
+            [schema.core :as schema]))
+
+(defn view-internal
+  [{:keys [type context? image network-image container-style]}]
+  [rn/view
+   {:style               (merge style/container container-style)
+    :accessibility-label :token-avatar}
+   [hole-view/hole-view
+    {:key   (hash context?)
```

Comment:
The hole is rendered dynamically based on the `context?` prop, and HoleView has an issue in Android(more info here: https://github.com/status-im/status-mobile/issues/15577) using the key prop for forceful re-render.

Issue screen recording:

https://github.com/status-im/status-mobile/assets/19279756/1649e6e6-5e22-47e4-8c0f-fb9e915be495

Reply:
If I understood correctly, it in an Android-specific workaround, maybe we can add a `platform/android?` check here.